src/components/OutputForm.tsx

```diff
@@ -144,5 +145,12 @@ export const OutputTime = ({ label, time, relative }: { label: string; time?: nu
     };
   });
 
-  return <OutputField label={label}>{time && <RelativeTimeText time={time} baseTime={nowTime} defaultToTime={!relative} />}</OutputField>;
+  let useRelative = relative ?? false;
+  if (relative == undefined && time) {
```

Comment:
The `relative` property should be an opt-in, not an opt-out.

I don't like this approach. Omitting the property should not mean it applies. For example, if we had a `bold` prop then `<OutputTime bold />` would make it bold and `<OutputTime />` would not be bold; we shouldn't have to do `<OutputTime bold=false />`.

A boolean prop is either present or it is not; present should be "do the thing" and not present should be "don't do the thing".